manager/manager.go

```diff
@@ -264,7 +264,12 @@ func (self *manager) Start() error {
 	}
 
 	// Setup collection of nvidia GPU metrics if any of them are attached to the machine.
-	self.nvidiaManager.Setup()
+	err = self.nvidiaManager.Setup()
+	if err != nil {
+		// We intentionally do nothing here. If Setup() fails then GetCollector()
+		// will return stats.NoopCollector
+		klog.Warningf("Setting up nvidia manager failed: %s", err)
```

Comment:
Since this is expected in some cases, can we just add something like: "NVidia GPU metrics will not be available"?  Every warning we add generates "Look at the warning I found; it caused ______!" bugs from users...

Reply:
We can leave this for another PR